python/py-pyopencl/Portfile

```diff
@@ -23,17 +23,18 @@ homepage            http://mathema.tician.de/software/${_name}
 master_sites        pypi:${_n}/${_name}/
 distname            ${_name}-${version}
 
-checksums           md5     bfd56711959bd4ca120a6f3fd8fc6aef \
-                    rmd160  df2ad4923eec06fea0b3055e1654ab173e99d7b3 \
-                    sha256  d2f7b04d2e819c6e90d6366b7712a7452a39fba218e51b11b02c85ab07fd2983
+checksums           md5     3cefdc1566100ef60b122686e003788f \
+                    rmd160  e848810e5f8911bd3a3b4620ea7c3330cf5d872a \
+                    sha256  7803f3128dbd28ae6f5b851a80ef586a35b9575406ea7bb068b8e1516f8043f0
```

Comment:
We are now adding the `size` parameter to `checksums`.

Reply:
and removing ```md5```

Reply:
done